src/Sylius/Behat/Context/Setup/OrderContext.php

```diff
@@ -170,6 +170,18 @@ public function thereIsCustomerThatPlacedOrder(CustomerInterface $customer, $ord
     }
 
     /**
+     * @Given a customer :customer added something to cart
+     */
+    public function customerStartedCheckout(CustomerInterface $customer)
```

Comment:
That's why the previous version of this step was ``Given a customer :customer started checkout`` 😄  Do we have any idea how to solve this problem? I would like not to add product to card as it's irrelevant in this scenario. /cc @pjedrzejewski